src/components/sidenav/sidenav.scss

```diff
@@ -44,6 +44,8 @@ md-sidenav {
   &.md-locked-open.md-closed.md-sidenav-left,
   &.md-locked-open.md-closed.md-sidenav-right,
   &.md-locked-open-remove.md-closed {
+    transition: width $swift-ease-in-duration $swift-ease-in-timing-function,
+                min-width $swift-ease-in-duration $swift-ease-in-timing-function;
```

Comment:
Instead of repeating it the third time, it would be better to create a SCSS variable at the top and just it for the three usages (makes changing way more straightforward)